README.md

```diff
@@ -51,12 +51,12 @@ documentation].
 
 | Release |  Current Patch  | Release Date |      EOL      |
 |:-------:|:---------------:|:------------:|:-------------:|
-|   v1.1  |     [v1.1.3]    |  Mar 3, 2021 |  August 2021  |
+|   v1.1  |     [v1.1.4]    | Aug 23, 2021 |  August 2021  |
```

Comment:
@AaronME we have actually been keeping these release dates as the first release for the given branch as a way to signify the duration between initial release and EOL. Do you think it makes more sense to update to match the latest patch date?